LoopKit/LoopPlugin.swift

```diff
@@ -0,0 +1,21 @@
+//
+//  LoopPlugin.swift
+//  LoopKit
+//
+//  Created by Pete Schwamb on 7/24/19.
+//  Copyright © 2019 LoopKit Authors. All rights reserved.
+//
+
+import Foundation
+
+public enum LoopPluginBundleKey: String {
```

Comment:
Presumably we'll need to add a ServiceManager version of these at some point, yes?

Reply:
Yes, I believe so.